.gitignore

```diff
@@ -63,6 +63,7 @@ regtests/ww3_tp2.17/input/inlet.msh
 regtests/ww3_tp2.21/input/mesh.msh
 regtests/ww3_tp2.21/input/obstructions_local.glo_unst.in
 regtests/ww3_tp2.21/input/obstructions_shadow.glo_unst.in
+regtests/ww3_tp2.14/input/oasis3-mct/lib/mct/*/.cvsignore
```

Comment:
I don't see where this file was renamed from .cvsignore to .gitignore or deleted (https://github.com/NOAA-EMC/WW3/blob/develop/regtests/ww3_tp2.14/input/oasis3-mct/lib/mct/.cvsignore) so I don't know how adding this to the gitignore is helpful?

Reply:
There are also a couple of other .cvsignore files in the regtests/ww3_tp2.14/input/oasis3-mct directories, maybe we just leave this out of this PR since it's not actually related and let @mickaelaccensi  clean this up in the future?